DSharpPlus.UnifiedCommands/Message/Internals/DefaultErrorHandler.cs

```diff
@@ -0,0 +1,102 @@
+using DSharpPlus.Entities;
+
+namespace DSharpPlus.UnifiedCommands.Message.Internals;
+
+/// <summary>
+/// The default class for error handling. 
+/// </summary>
+internal class DefaultErrorHandler : IErrorHandler
+{
+    public async Task HandleConversionAsync(InvalidMessageConversionError error, DiscordMessage message)
+    {
+        DiscordMessageBuilder msgBuilder = new();
+        msgBuilder.WithReply(message.Id);
+        string argumentName = error.Name.Count() == 0 ? string.Empty : $" `{error.Name}`";
+
+        msgBuilder.WithAllowedMentions(Mentions.None);
+
+        // This can be changed to a if else for .IsPositionalArgument
+        switch (error.Type)
+        {
+            case InvalidMessageConversionType.NotAValidNumber:
+                if (error.IsPositionalArgument)
+                {
+                    msgBuilder.WithContent(
+                        $"You cannot convert value `{error.Value}` to a number for argument{argumentName}");
```

Comment:
It is not missing a space. The value `argumentName` has a space in it. This is so if it is a nameless argument (should never happen, this is for edge cases.). It'll look like `for argument.` instead of `for argument .`